partitioned-table.md

```diff
@@ -1025,6 +1025,16 @@ ALTER TABLE members OPTIMIZE PARTITION p0;
 ERROR 8200 (HY000): Unsupported optimize partition
 ```
 
```

Comment:
@mjonss please also add a section in this PR to describe how to modify one partition type to another. Thanks.

Reply:
I rephrased 'Convert a non-partitioned table to a partitioned table' into 'Partition an existing table' with some clarifying text.